error/src/lib.rs

```diff
@@ -577,11 +577,17 @@ impl CaliptraError {
     pub const KAT_HMAC384_FAILURE: CaliptraError = CaliptraError::new_const(0x90030001);
     pub const KAT_HMAC384_TAG_MISMATCH: CaliptraError = CaliptraError::new_const(0x90030002);
 
+    /// Deprecated
     pub const KAT_ECC384_SIGNATURE_GENERATE_FAILURE: CaliptraError =
```

Comment:
If these are not used, let's remove them.

Reply:
I can do that. I just left them in case someone was referencing this documentation in in main when they are working on an older version. You do agree with "retiring" those values though, right? I don't think we want to reuse an old error code if we deprecate it.

Reply:
In [this](https://github.com/chipsalliance/caliptra-sw/pull/1767/files#diff-009b2077ef4a38a7715447e771702e14e3f522198f0a8cc09f073a08bd02eaa8) file from https://github.com/chipsalliance/caliptra-sw/pull/1767 I removed error codes but left comments to indicate what the codes used to be used for, to prevent reuse.

Reply:
I think that's a good idea. I'll make that change.